Screen-Sharing/app/build.gradle

```diff
@@ -25,4 +25,5 @@ dependencies {
     compile 'pub.devrel:easypermissions:0.2.0'
```

Comment:
I will add this as a github issue. This is a valid point, but I want to push current updates today.